server/app.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import logger from 'morgan';
+
+const app = express();
+const http = require('http');
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({
+  extended: true,
+}));
+
+
+app.all('*', (req, res) => {
+  res.status(404).send('The page you are looking for was not found');
```

Comment:
Good work @Raymond-Osy However, Please check the project doc for more clarity on the format of the response message.

Reply:
Alright, Noted